docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+
+First, let's retrieve the IP address of the `nginx` pods:
+
+```bash
+LOCAL_POD_IP=$(kubectl get pod nginx-local -n liqo-demo --template={{.status.podIP}})
+REMOTE_POD_IP=$(kubectl get pod nginx-remote -n liqo-demo --template={{.status.podIP}})
+echo "Local Pod IP: ${LOCAL_POD_IP} - Remote Pod IP: ${REMOTE_POD_IP}"
+```
+
+You can fire up a pod and run `curl` from inside the cluster:
+
+```bash
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${LOCAL_POD_IP}
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${REMOTE_POD_IP}
+```
+
+Both commands should lead to a successful outcome (i.e., return a demo web page), regardless of whether each pod is executed locally or remotely.
+
+#### Expose the pods through a Service
+
+The above configuration creates a Service which is designed to serve traffic to the previously deployed pods.
+This is a traditional [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/) and can work with Liqo with no modifications.
+
+Indeed, inspecting the Service, it is possible to observe that both `nginx` pods are correctly specified as endpoints.
+Nonetheless, it is worth noticing that the first endpoint (i.e. `10.200.1.10:80` in this example) refers to a pod running in the _local_ cluster, while the second one (i.e. `10.202.1.10:80`) points to a pod hosted by the _remote_ cluster.
+
+```bash
+kubectl describe service liqo-demo -n liqo-demo
+```
+
+```text
+Name:              liqo-demo
+Namespace:         liqo-demo
+Labels:            <none>
+Annotations:       <none>
+Selector:          app=liqo-demo
+Type:              ClusterIP
+IP Family Policy:  SingleStack
+IP Families:       IPv4
+IP:                10.93.84.150
+IPs:               10.93.84.150
+Port:              web  80/TCP
+TargetPort:        web/TCP
+Endpoints:         10.200.1.10:80,10.202.1.10:80
+Session Affinity:  None
+Events:            <none>
+```
+
+#### Check the Service connectivity
+
+It is now possible to contact the Service: as usual, Kubernetes will forward the HTTP request to one of the available back-end pods.
+Additionally, all traditional mechanisms still work seamlessly (e.g. DNS discovery), even though one of the pods is actually running in a _remote_ cluster.
+
+You can fire up a pod and run `curl` from inside the cluster:
+
+```bash
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- \
+    curl --silent liqo-demo.liqo-demo.svc.cluster.local | grep 'Server'
+```
+
+```{admonition} Note
+Executing the previous command multiple times, you will observe that part of the requests are answered by the pod running in the *local* cluster, and in part by that in the *remote* cluster (i.e., the Server value changes).
```

Comment:
```suggestion
Executing the previous command multiple times, you will observe that part of the requests are answered by the pod running in the *local* cluster, and in part by that in the *remote* cluster (i.e., the `Server` value changes).
```

Reply:
This section is removed following Marco Iorio's suggestions.